src/AxeWindows.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.757
+# Visual Studio Version 16
```

Comment:
This is stamped for VS2019. Should we make 2019 our official build environment? I like the plan, but it feels like it should be a separate PR (and update the docs that reference VS2017)